src/libraries/Microsoft.PowerFx.Core/Parser/ParseFormulasResult.cs

```diff
@@ -139,5 +139,13 @@ public UDFArg(IdentToken nameIdent, IdentToken typeIdent, Token colonToken, int
             ArgIndex = argIndex;
             ColonToken = colonToken;
         } 
+    }
+
+    internal enum UserDefinitionType
```

Comment:
>UserDefinitionType [](http://example.com/codeflow?start=18&length=18)

This enum should be defined closer to where it is used - either in UserDefinitionInfo.cs or in its own file